components/base/kinematicbase/fake_kinematics.go

```diff
@@ -203,7 +199,7 @@ func WrapWithFakePTGKinematics(
 	}
 	ptgs := ptgProv.PTGSolvers()
 	traj := motionplan.Trajectory{{frame.Name(): zeroInput}}
-	path := motionplan.Path{{frame.Name(): referenceframe.NewPoseInFrame(origin.Parent(), spatialmath.Compose(origin.Pose(), sensorNoise))}}
+	path := motionplan.Path{{frame.Name(): origin}}
```

Comment:
Do we no longer want to consider sensor noise for fake mockups?

Reply:
I think with this change we are applying the same sensor noise twice, since it is also applied in the Localizer